buildscripts/azure/azure-windows.yml

```
@@ -34,7 +34,7 @@ jobs:
         call conda remove --all -q -y -n %CONDA_ENV%
         call conda create -n %CONDA_ENV% -q -y python=%PYTHON% cmake
         call activate %CONDA_ENV%
-        call conda install -y -q -c numba/label/dev llvmdev="14.*" libxml2
+        call conda install -y -q --override-channels -c gmarkall llvmdev="14.*" libxml2
```

Not sure if the build picked up llvmdev from the right place given the failures - maybe worth trying:

```suggestion
        call conda install -y -q gmarkall::llvmdev libxml2
```